### PR TITLE
update initial setup steps

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

**Comment by ehelms** [comment]
Looks like this is tackling http://projects.theforeman.org/issues/9236 and I'd say it might be better to link to the documentation in the bastion README instead of replicating it.

**Comment by thomasmckay** [comment]
I can add a link to the doc at this point like, "For full details on developing Bastion go _here_". However, to force a dev to click through a link for the one command they need to run seems unnecessary to me.

File: developers/bastion/index.md [hunk 1]
```diff
@@ -77,14 +77,20 @@ nvm use 0.10
...
 ### Installing Local Packages
 
-Ensure you are in the engines/bastion directory or change to the directory.
+Ensure you are in the engines/bastion_katello directory or change to the directory.
 Install the local node modules defined within package.json:
 
 ```bash
-cd ~/path/to/mykatello/engines/bastion
+cd ~/path/to/mykatello/engines/bastion_katello
 npm install
```
Comment (thomasmckay) [inline developers/bastion/index.md]: @ehelms - What do this step do? Even after bastion:setup, npm install fails for me. I can run 'grunt ci' though.

Comment (ehelms) [inline developers/bastion/index.md]: This should/could just be deleted. Most of what is in this file should be gotten from the Bastion README

**Pull request closed without merging**